src/components/adminWaiverEdit.vue

```diff
@@ -0,0 +1,395 @@
+      </small>
+      <small class="col-sm-12 text-left">
+        Preview any changes on the waiver below.
+      </small>
+    </div>
+    <div ref="updateWaiver" class="row col-sm-12">
+      <div class="row col-sm-11 text-left my-5">
+        <form v-on:submit.prevent="update_waiver" class="form">
+          <table class="table text-left my-3">
+            <tbody>
+              <input-table-row :label="waiver_labels.title" type="text" 
+                v-model="waiver.title" 
+                :min-length="waiver_validators.title_min"
+                v-on:invalid="invalid.title = $event">
+              </input-table-row>
+              <tr>
+                <input-label-table-large v-bind:class="{ 'has-error': headers_have_errors }"
+                  :label="waiver_labels.header">
+                </input-label-table-large>
+                <td colspan="2" class="col-sm-10 text-left align-baseline">
+                  <input-row-no-label type="text" 
+                    :label="waiver_labels.header"
+                    v-model="waiver.header[0]" 
+                    :min-length="waiver_validators.header_min"
+                    v-on:invalid="headerValidator($event, 0)">
+                  </input-row-no-label>
+                  <div class="row col-sm-12">
+                    DATE
+                    <span class="mx-2" v-if="type === 'volunteer'">
+                      (date) by VOLUNTEER NAME
+                    </span>
+                  </div>
+                  <input-row-no-label type="textarea" 
+                    :label="waiver_labels.header"
+                    v-model="waiver.header[1]" 
+                    :min-length="waiver_validators.header_min"
+                    v-on:invalid="headerValidator($event, 1)">
+                  </input-row-no-label>
+                  <div v-if="type === 'camper'" >
+                    <span class="row col-sm-12">CHILD'S NAME</span>
+                    <input-row-no-label type="textarea" 
+                      :label="waiver_labels.header"
+                      v-model="waiver.header[2]" 
+                      :min-length="waiver_validators.header_min"
+                      v-on:invalid="headerValidator($event, 2)">
+                    </input-row-no-label>
+                    <span class="row col-sm-12">PARENT'S NAME</span>
+                    <input-row-no-label type="textarea" 
+                      :label="waiver_labels.header"
+                      v-model="waiver.header[3]" 
+                      :min-length="waiver_validators.header_min"
+                      v-on:invalid="headerValidator($event, 3)">
+                    </input-row-no-label>
+                  </div>
+                </td>
+              </tr>
+            <tr>
+              <input-label-table-large 
+                v-bind:class="{ 'has-error': items_have_errors }"
+                :label="waiver_labels.items">
```

Comment:
can you move titles like "legal items" to their own lines? Asking because the width of the text sections is being scrunched a lot, with most of the screen around them being blank

Reply:
yes - that's a good idea
i modified the spacing to add them to their own line